src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -10853,6 +10853,18 @@ instruct rep_stos_large(rcx_RegL cnt, rdi_RegP base, regD tmp, rax_RegI zero,
   ins_pipe(pipe_slow);
 %}
 
+instruct rep_stos_im(immL cnt, rRegP base, regD tmp, rRegI zero, Universe dummy, rFlagsReg cr)
```

Comment:
What about x86_32.ad?

Reply:
Hi Vladimir, Pattern has been added in x86_32.ad